### PR TITLE
Fix lldb shared lib loading processing.

### PR DESCRIPTION
**Pull request by paulmaybee**
Changes to be committed:
	modified:   src/MICore/MICommandFactory.cs
	modified:   src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

### CODE REVIEW

File: src/MICore/MICommandFactory.cs [hunk 1]
```diff
@@ -617,16 +617,6 @@ public override bool SupportsStopOnDynamicLibLoad()
             return false;
         }
 
-        public override Results IsModuleLoad(string cmd)
```
Comment (gregg-miskelly) [inline src/MICore/MICommandFactory.cs]: Should we make the base method non-virtual now that we don't have any overrides of it?

File: src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs [hunk 2]
```diff
@@ -101,23 +101,33 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                         _waitDialog.ShowWaitDialog(file);
                     }
                 }
-                else if (results.Results.Contains("shlib-info")) //lldb
+                else if (this.MICommandFactory.Mode == MIMode.Clrdbg)
```
Comment (gregg-miskelly) [inline src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs]: NOTE: Now that the lldb library load event is more applicable, we should update clrdbg to match lldb. I added a cleanup item to the clrdbg list.

**Comment by gregg-miskelly** [comment]
:+1:

File: src/MICore/MICommandFactory.cs (hunk 1 above)
Comment (paulmaybee) [inline src/MICore/MICommandFactory.cs]: done

**Pull request merged**